### ISSUE TITLE
Work out how to host docs for individual tools 

### ISSUE THREAD

**Issue by adamltyson** [brainglobe/new-website#4]
Ideally we want:
a) Tool-specific docs to be part of the relevant repo so that docs can be updated along with code (and autogenerated API docs)
b) All docs to be accessible via one site (brainglobe.info)

How do we do this? I haven't found any tool to automate this, the best way seems to be some kind of custom build script. The process could be something like:
* Include docs in all repos
* Have a central repo (this one) which contains general BrainGlobe info, and a script to:
    * Find all repos with docs
    * Copy files
    * Build one large site with all appropriate links

Any ideas @alessandrofelder? We should think about this as we start to accumulate more documentation for different parts of the project.

**Comment by thisisrick25** [comment]
Hi @adamltyson, are there any constraints on what tech stack should be used to build the website and aggregate the docs?

**Comment by adamltyson** [comment]
This issue is slightly outdated, we have a single website to host all the docs (this is the repo). What's missing is a way to aggregate the API docs (to be built from the docstrings using Sphinx) into the one site.

**Comment by adamltyson** [comment]
Yep, that's what we need. The tricky bit is doing it for every BrainGlobe package, and rendering them on a single website.

**Comment by thisisrick25** [comment]
I think I figured it out. I will raise a PR once I'm done with the remaining BG packages.
